clang/include/clang/Basic/MacroBuilder.h

```diff
@@ -26,8 +27,14 @@ class MacroBuilder {
   MacroBuilder(raw_ostream &Output) : Out(Output) {}
 
   /// Append a \#define line for macro of the form "\#define Name Value\n".
-  void defineMacro(const Twine &Name, const Twine &Value = "1") {
+  /// If DeprecationMsg is provided, also append a pragma to deprecate the
+  /// defined macro.
+  void defineMacro(const Twine &Name, const Twine &Value = "1",
+                   std::optional<Twine> DeprecationMsg = std::nullopt) {
```

Comment:
Just use Twine with a default ""? Shouldn't have a value twine optional

Reply:
Changed in a798615ab4608f1681b15a526bc898df3c12bfa3.